### DISCUSSION TITLE
Android 备用机短信转发 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by SF-Zhou** [SF-Zhou/sf-zhou.github.io#180] [Announcements]
https://sf-zhou.github.io/productivity/android_sms_to_dingtalk.html

**Comment by silentcrow09** [comment]
挺好的

**Comment by JosephAU** [comment]
感谢，很有价值

**Comment by zihluwang** [comment]
作者大大，请问这个Notice Token是来自于哪儿啊？还有not notice token来自于哪儿啊？
![pic](http://publicfiles.cdn.vorbote.cn/dingTalkRobot11281831.png)

**Comment by SF-Zhou** [comment]
> @TheodoreHills
> 作者大大，请问这个Notice Token是来自于哪儿啊？还有not notice token来自于哪儿啊？

是我自己拉的两个机器人，一个在钉钉里设置为不通知。
